docs/TOFS_pattern.rst

```diff
@@ -0,0 +1,443 @@
+.. _tofs_pattern:
```

Comment:
Since we're adding this document, let's add `libtofs.jinja` as well.

Reply:
Not to mention, update `pillar.example` with the `tofs` section from `template-formula`.  Or would this cause some confusion?  Should we include any of the TOFS stuff at all, until the formula is adapted for TOFS?

Reply:
I generally copy the doc because I copy all docs directory, but nothing more.

Reply:
I think its better to copy `libtofs.jinja` and edit `pillar.example` only when implementing TOFS, so that you have latest versions from `template-formula`.

Reply:
So perhaps it's best to leave out the document as well, since that will change if `libtofs.jinja` changes.  And there is a chance of that due to https://github.com/saltstack-formulas/template-formula/pull/86.